client/src/components/StatsList/StatButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { Stat } from "../../types/card/card.types";
+
+interface StatButtonProps {
+  stat: Stat;
+  playerId: string;
+  playRound: (stat: Stat, playerId: string) => void;
+}
+export const StatButton: React.FC<StatButtonProps> = ({
+  stat,
+  playerId,
+  playRound,
+}) => {
+  const handleButtonClick = () => {
+    playRound(stat, playerId);
+  };
+  return (
+    <button data-testid="stat-button" onClick={handleButtonClick}>
```

Comment:
Instead of "playRound" you could call the prop here something like "handleButtonClick" or "handleStatChosen" then inline the function directly in the onClick like so:

```<button data-testid="stat-button" onClick={() => handleButtonClick}>```